dingo/pipe/parser.py

```diff
@@ -639,37 +639,36 @@ def create_parser(top_level=True, usage=None):
         help="Memory allocation request (GB) for data generation step",
     )
     submission_parser.add(
-        "--request-memory-sampling",
+        "--request-memory",
         type=float,
-        default=128.0,
-        help="Memory allocation request (GB) for sampling step. Default is 128GB",
+        default=8.0,
+        help="Memory allocation request (GB) for sampling step. Default is 8GB",
+    )
+    submission_parser.add(
+        "--request-cpus",
+        type=int,
+        default=1,
+        help=(
+            "Use multi-processing during sampling. This options sets the number of cores to "
+            "request per job when performing sampling."
+        ),
     )
     submission_parser.add(
         "--request-memory-importance-sampling",
         type=float,
-        default=128.0,
-        help="Memory allocation request (GB) for sampling step. Default is 128GB"
+        default=8.0,
+        help="Memory allocation request (GB) for importance sampling step. Default is 32GB"
```

Comment:
Can you adopt the help information to be consistent with the default value?
```suggestion
        help="Memory allocation request (GB) for importance sampling step. Default is 8GB"
```